states/massachusetts/elections.toml

```diff
@@ -8,7 +8,87 @@ date = 2020-09-01
 name = "Congressional Primary"
 original_date = 2020-09-01
 
+[20200901.registration]
+in_person_by = 2020-08-22
+online_by = 2020-08-22T23:59:00
+postmarked_by = 2020-08-22
+sources = [
+  "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
+  "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
+]
+# Note the Day to Register was extended to 10 days from 20 days prior in 2020
+
+[20200901.absentee.application]
+email_by = 2020-10-31T12:00:00
+fax_by = 2020-10-31T12:00:00
+received_by = 2020-10-31T12:00:00
+# Because you can vote absentee in-person day before is OK
+sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"]
+
+[20200901.absentee]
+in_person_by = 2020-09-01 # You must make an appointment
+received_by = 2020-09-01 
+sources = ["http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm"] 
+
+[20200901.poll.early.application]
+# Mail in Early Voting Application
+email_by = 2020-08-26
+fax_by = 2020-08-26
+in_person_by = 2020-08-26
+received_by = 2020-08-26
+sources = ["https://www.sec.state.ma.us/ele/eleev/early-voting-by-mail.htm"]
+
+[20200901.poll.early]
+in_person_by = 2020-08-28
+in_person_start = 2020-08-22
+received_by = 2020-09-01T20:00:00
+sources = ["https://www.sec.state.ma.us/ele/eleev/evidx.htm"]
+
+[20200901.poll]
+in_person_by = 2020-09-01T20:00:00
+in_person_start = 2020-09-01T07:00:00
+
 [20201103]
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+in_person_by = 2020-10-24
+online_by = 2020-10-24T23:59:00
+postmarked_by = 2020-10-24
+sources = [
+  "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
+  "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
+]
+# Note the Day to Register was extended to 10 days from 20 days prior in 2020
+
+[20201103.absentee.application]
```

Comment:
Overseas absentee has 10 days to arrive. From the FAQ: "If you mail your ballot from outside the country, it has until November 13th to arrive, but it must still be postmarked by Election Day."

Reply:
I'm not covering overseas or military all. I've skipped it for every state.